build-config/make/mtdutils.make

```diff
@@ -9,7 +10,8 @@
 # This is a makefile fragment that defines the build of mtdutils
 #
 
-MTDUTILS_VERSION	= 1.5.0
+MTDUTILS_VERSION	?= 1.5.0
+MTDUTILS_LATEST_RELEASE_VERSION = 1.5.2
```

Comment:
please upload the latest mtdutils commit tar mtd-utils-e4c8885.tar.gz to onie server